cg/apps/vogue.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
If you save as uft-8, the file is utf-8 ;)

```suggestion

```

Reply:
Unless it has another purpose?

Reply:
👍